website/validate-token.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015 - 2018 Uber Technologies, Inc.
```

Comment:
Nit: Should this go into a scripts folder.

Reply:
👍 either `bin` or `scripts`